ui/css/utilities/colors.scss

```diff
@@ -4,6 +4,8 @@
   --mainnet: #29b6af;
   --inherit: inherit;
   --transparent: transparent;
+  --color-network-linea-testnet-default: #000000;
+  --color-network-linea-testnet-inverse: #fcfcfc;
   // DO NOT CHANGE
```

Comment:
Usually we wouldn't allow static hex values but I think we can refactor the test networks to use images like other token icons during the multichain v0.5 work so this will be temporary.